# signals permission to start referencing operation
# setting a timeout
int16 timeout
---
bool done